From: Dan Carpenter <dan.carpenter@oracle.com>
Date: Thu, 10 Aug 2017 12:56:14 +0300
Subject: hns3pf: Fix some harmless copy and paste bugs
Patch-mainline: v4.14-rc1
Git-commit: 720a8478b2689bd221e4a8744e1695de7966a22f
References: bsc#1104353 FATE#326415

These were copy and paste bugs, but I believe they are harmless.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/hisilicon/hns3/hns3pf/hns3_enet.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/net/ethernet/hisilicon/hns3/hns3pf/hns3_enet.c
+++ b/drivers/net/ethernet/hisilicon/hns3/hns3pf/hns3_enet.c
@@ -330,7 +330,7 @@ static int hns3_nic_mc_sync(struct net_d
 	struct hns3_nic_priv *priv = netdev_priv(netdev);
 	struct hnae3_handle *h = priv->ae_handle;
 
-	if (h->ae_algo->ops->add_uc_addr)
+	if (h->ae_algo->ops->add_mc_addr)
 		return h->ae_algo->ops->add_mc_addr(h, addr);
 
 	return 0;
@@ -342,7 +342,7 @@ static int hns3_nic_mc_unsync(struct net
 	struct hns3_nic_priv *priv = netdev_priv(netdev);
 	struct hnae3_handle *h = priv->ae_handle;
 
-	if (h->ae_algo->ops->rm_uc_addr)
+	if (h->ae_algo->ops->rm_mc_addr)
 		return h->ae_algo->ops->rm_mc_addr(h, addr);
 
 	return 0;
